tests/integration/states/ssh.py

```diff
@@ -72,7 +72,8 @@ def test_present(self):
 
         # save twice, no changes
         ret = self.run_state('ssh_known_hosts.present', **kwargs)
-        self.assertSaltStateChangesEqual(ret, {})
+        # This will change the key, the second one is the true test
+        self.assertTrue(ret, {})
```

Comment:
That isn't the behaviour I want. :]

I only want to assert that the dictionary isn't empty. In reality the assert can be removed entirely since the following assert L80 is what matters.

But, yes, there are far better ways to do this. I'll just remove the assert since it's not important really.